OpenSim/Simulation/Model/Model.cpp

```diff
@@ -1428,87 +1428,47 @@ void Model::removeController(Controller *aController)
 
 
 
-//==========================================================================
+//==============================================================================
 // OPERATIONS
-//==========================================================================
-//--------------------------------------------------------------------------
+//==============================================================================
+
+//------------------------------------------------------------------------------
 // SCALE
-//--------------------------------------------------------------------------
-//_____________________________________________________________________________
-/**
- * Scale the model
- *
- * @param aScaleSet the set of XYZ scale factors for the bodies
- * @param aFinalMass the mass that the scaled model should have
- * @param aPreserveMassDist whether or not the masses of the
- *        individual bodies should be scaled with the body scale factors.
- * @return Whether or not scaling was successful.
- */
-bool Model::scale(SimTK::State& s, const ScaleSet& aScaleSet, double aFinalMass, bool aPreserveMassDist)
+//------------------------------------------------------------------------------
+bool Model::scale(SimTK::State& s, const ScaleSet& scaleSet, double finalMass,
```

Comment:
The compile issue may be due to the fact that `finalMass` and `preserveMassDist` have default arguments so the default signature is the same as the virtual `ModelComponent::scale`.